packages/flutter_fuchsia/lib/example_main.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:io';
+import 'dart:async';
+import 'dart:core';
+
+import 'package:logging/logging.dart';
+
+import 'flutter_fuchsia.dart';
+
+Future<Null> main(List<String> args) async {
```

Comment:
Let's add some dartdocs clarifying why this example is here and what it is demonstrating.

Reply:
Done.